### PR TITLE
Dias scoring tests

### PR DESCRIPTION
**Pull request by tttppp**

### CODE REVIEW

**Comment by tttppp** [comment]
Note that several of the tests currently fail and have been commented out. These are for discussion before merging.

File: game/game_result.go [hunk 1]
```diff
@@ -62,10 +62,14 @@ func (gs GameScores) Assign() {
 	minTopperSize := int(math.Ceil(float64(numSCs)/float64(len(gs))) + 1)
 	// Score per SC is 34 / number of SCs.
 	scorePerSC := 34.0 / float64(numSCs)
+	survivalPart := 66.0 / float64(survivors)
 	// Tribute is one for each SCs over minimum topper size.
 	tributePerSurvivor := 0.0
 	if topperSize > minTopperSize {
 		tributePerSurvivor = scorePerSC * float64(topperSize-minTopperSize)
+		if tributePerSurvivor > survivalPart {
```
Comment (zond) [inline game/game_result.go:70]: When this is corrected, the check if the survivor has more points than the tribute part is redundant and should be removed, right?

Comment (zond) [reply game/game_result.go:70]: I took the liberty of removing the redundant case, the tests passed and it made sense.

Comment (tttppp) [reply game/game_result.go:70]: Thanks!

**Comment by tttppp** [comment]
Ready for merge assuming the build passes :+1:

**Pull request merged**